predict/build.sbt

```diff
@@ -0,0 +1,9 @@
+name := "geotensorflow-predict"
+
+libraryDependencies ++= Seq(
+  "org.locationtech.geotrellis" %% "geotrellis-spark"      % Version.geotrellis,
+  "org.locationtech.geotrellis" %% "geotrellis-s3"         % Version.geotrellis,
+  "org.apache.spark" %% "spark-core"    % Version.spark,
+  "org.apache.hadoop" % "hadoop-client" % Version.hadoop,
+  "org.scalatest"    %% "scalatest"     % Version.scalaTest % "test"
+)
```

Comment:
Assuming this will be an evolving list, but might make sense to clear/use only what's necessary here as to not have things hanging around that don't need to be.

Reply:
sounds good. so remove all libraryDependencies or are there any that will be necessary?